src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,8 +1,36 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
 public class HelloWorld {
+    public double calculate(double value1, double value2, char operation)
+            throws BadOperandException {
 
+        double result;
+
+        switch (operation) {
+            case ('*'):
+                result = value1 * value2;
```

Comment:
```suggestion
                return value1 * value2;
```
you don't really need `result` variable

Reply:
done